pkgs/development/libraries/libiio/default.nix

```
@@ -50,6 +52,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DUDEV_RULES_INSTALL_DIR=${placeholder "out"}/lib/udev/rules.d"
+    "-DPython_EXECUTABLE=${buildPackages.python3}/bin/python"
```

```suggestion
    "-DPython_EXECUTABLE=${python.pythonForBuild.interpreter}"
```

so there's no version mismatch